Update CloneFieldsControl.cs

OMGUSED opened the pull request:
Bug Fix: Removed renaming of OptionSet for attributes that use global OptionSets

OMGUSED committed:
Update CloneFieldsControl.cs

Bug Fix: Removed renaming of OptionSet for attributes that use global OptionSets

DigitalFlow commented:
Hi @OMGUSED,

is this something that produced a bug in your system?

It seems that this might lead to Global Option Sets being duplicated in the system.

@harryremon: It seems this was introduced as a bug fix? [Fixed a bug that caused attributes of type "Global Optionset" fail to create](https://github.com/XRM-OSS/XrmToolBox.CloneFieldDefinitions/commit/e08afb5d96a1fdfce3a8850da11ab4b401d6f8bc#diff-f7afba9b5bc03e74f53c54d0f701e82521fc107f0db401bdd214c019b30a242e).
Can you still remember which bug that fixed?

Kind regards,
Florian

OMGUSED commented:
I tried to use the tool to copy fields that use global optionsets on a V9
on-premise. The copy kept failing because it kept trying to look for a
global optionset that didn't exist. After removing the extra code the copy
was successful and re-used the existing global optionset.

I think globals should be reused instead of copied to a new one

On Mon, 28 Jun 2021, 09:34 Florian Krönert, ***@***.***>
wrote:

> Hi @OMGUSED <https://github.com/OMGUSED>,
>
> is this something that produced a bug in your system?
>
> It seems that this might lead to Global Option Sets being duplicated in
> the system.
>
> @harryremon <https://github.com/harryremon>: It seems this was introduced
> as a bug fix? Fixed a bug that caused attributes of type "Global
> Optionset" fail to create
> <https://github.com/XRM-OSS/XrmToolBox.CloneFieldDefinitions/commit/e08afb5d96a1fdfce3a8850da11ab4b401d6f8bc#diff-f7afba9b5bc03e74f53c54d0f701e82521fc107f0db401bdd214c019b30a242e>
> .
> Can you still remember which bug that fixed?
>
> Kind regards,
> Florian
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/XRM-OSS/XrmToolBox.CloneFieldDefinitions/pull/20#issuecomment-869441918>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACM4EKNGE345L7YDFCYRZ5TTVAQYXANCNFSM453MCRBA>
> .
>

Commit:
Renamed Comboboxes, Changed style to Dropdown

Renamed comboboxes and changed their style to allow the user to type the logical name so the list will show the next item that matches what the user typed. Didn't use the second style due to conflict in the UI (it would show both its source list and the edit suggestions)

Commit:
Updated Packages, changed to .NET 4.8 to match latest XTB
